### PR TITLE
Fix merge section in dark theme

### PR DESCRIPTION
**Pull request by lafriks**
Before:

image

After:

image

### CODE REVIEW

File: web_src/less/themes/theme-arc-green.less [hunk 1]
```diff
@@ -675,18 +655,41 @@ a.ui.basic.green.label:hover {
     color: #fff !important;
 }
 
-.repository.view.issue .comment-list .comment .content .header:after {
-    border-right-color: #404552;
+.repository.view.issue .comment-list .comment .content {
+    > .bottom.segment {
+        background: #353945;
+        a {
+            border: solid 1px #353945;
```
Comment (silverwind) [inline web_src/less/themes/theme-arc-green.less:976]: Can this just be `border-color`?

File: web_src/less/themes/theme-arc-green.less [hunk 2]
```diff
@@ -675,18 +655,41 @@ a.ui.basic.green.label:hover {
...
+            background-color: #353945;
+        }
+    }
+
+    .header {
+        color: #dbdbdb;
+        background-color: #404552;
+        border-bottom: 1px solid #353944;
```
Comment (silverwind) [inline web_src/less/themes/theme-arc-green.less:984]: `border-top-color`?

File: web_src/less/themes/theme-arc-green.less [hunk 3]
```diff
@@ -675,18 +655,41 @@ a.ui.basic.green.label:hover {
...
+    }
+
+    .merge-section {
+        background-color: #404552;
+        border-top: 1px solid #353944;
```
Comment (silverwind) [inline web_src/less/themes/theme-arc-green.less:989]: `border-top-color`?

**Comment by lafriks** [comment]
@silverwind these values was there already so I would not like to change them (I just moved them into one group). Only actual changes are added `> .merge-section.no-header:(before|after)`

**Review by silverwind — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by lafriks** [comment]
:rocket:

**Comment by zeripath** [comment]
Please send backport

**Pull request merged**